mbuild/utils/geometry.py

```diff
@@ -55,3 +55,28 @@ def coord_shift(xyz, box):
         xyz += box_max
 
     return xyz
+
+def wrap_coords(xyz, box):
+    """ Wrap coordinates inside box
+
+    Parameters
+    ---------
+    xyz : numpy.array of points with shape N x 3
+    box : numpy.array specifing the size of box ie [Lx, Ly, Lz]
+
+    Returns
+    -------
+    xyz : numpy.array of points with shape N x 3
+
+    Notes
+    -----
+    Assumes we are wrapping inside the positive octant
+    """
+    for atom_xyz in xyz:
+        if atom_xyz[0] < 0: atom_xyz[0]+= box[0]
```

Comment:
At the moment this doesn't consider if the particle is displaced more than one period away from the box, which they usually are in my unwrapped hoomd trajectories. I think this would a pretty easy thing to add to this PR with something like this:
`if atom_xyz[0] < 0: atom_xyz[0] += int(atom_xyz[0] / box[0]) * box[0]`